environments/pyyaml/environment.yml

```diff
@@ -2,6 +2,5 @@ name: pyyaml
 channels:
   - conda-forge
   - bioconda
-  - defaults
```

Comment:
Do we need to add `nodefaults` too to be extra sure? https://stackoverflow.com/questions/67695893/how-do-i-completely-purge-and-disable-the-default-channel-in-anaconda-and-switch

Reply:
sounds good 🥳